lib/vast/Translation/FromSource.cpp

```diff
@@ -60,7 +60,7 @@ namespace vast::hl
         if (id_meta_flag) {
             cg::CodeGenWithMetaIDs(cgctx).emit_module(ast.get());
         } else {
-            cg::DefaultCodeGen(cgctx).emit_module(ast.get());
+            cg::DefaultCodeGen<cg::CodeGenContext>(cgctx).emit_module(ast.get());
```

Comment:
This should be deducible from constructor parameter. Add deduction guide if necessery.